CRM/Event/BAO/Participant.php

```diff
@@ -2122,63 +2122,60 @@ public static function recordAdjustedAmt($updatedAmount, $paidAmount, $contribut
     $pendingAmount = CRM_Core_BAO_FinancialTrxn::getBalanceTrxnAmt($contributionId);
     $pendingAmount = CRM_Utils_Array::value('total_amount', $pendingAmount, 0);
     $balanceAmt = $updatedAmount - $paidAmount;
-    if ($paidAmount != $pendingAmount) {
-      $balanceAmt -= $pendingAmount;
-    }
```

Comment:
@pradpnayak any objections of not deducting `$pendingAmount` from `$balanceAmt` ? As I think it was added with some intention in https://github.com/civicrm/civicrm-core/commit/18fdfcc3d52d25833467d834f25471f1ae5b80ec#diff-d05dcb57339ac128d06193a57b9ca1bcR2056.